src/mixins/raster-layer-line-mixin.js

```diff
@@ -94,8 +94,22 @@ function getTransforms(
     return state.data.length > 1
   }
 
+  const groupbyDim = state.transform.groupby ? state.transform.groupby.map((g, i) => ({
+    type: "project",
+    expr: `${state.data[0].table}.${g}`,
+    as: `key${i}`
+  })) : {}
```

Comment:
It seems like groupbyDim (and thus groupby below) can be an object in some cases, yet we're checking the `.length` of it below. This is the only part that seems a little odd. Should this be `[]` instead, perhaps?

Reply:
@jrajav, good catch. I changed. Thank you :)